### PR TITLE
Removes BtR-Extensions section

### PR DESCRIPTION
**Pull request by bbranan**
As noted in https://github.com/dpscollaborative/btr_bagit_profile/pull/2, the ability to search specific tags in a system that happens to ingest a bag doesn't seem to fit as part of the bag profile spec as it has nothing to do with the transfer of data.

The list of recommended tags duplicates the `"recommended": true` property associated with the individual tags. Given that this doesn't provide any additional information, and future edits may only update one of the two locations (causing the two data sources to become out-of-sync), it seems best to leave this out.

Dropping this section also means that we don't tie anything in this profile to the "BtR" grant by name, which seems desirable for longevity.

### CODE REVIEW

**Comment by diamondap** [comment]
I agree that the BtR extensions section should be removed.

**Comment by tdilauro** [comment]
@bbranan Your logic makes sense to me. I accept that it should be removed.

**Review by tdilauro — APPROVED** [review]
I agree with the logic here and approve of the change.

**Comment by bbranan** [comment]
@tdilauro or @diamondap - would one of you be willing to merge this one? I'm assuming we're following the "don't merge your own PRs" approach.

**Pull request merged**
